src/applications/hca/tests/unit/components/FormPages/DisabilityConfirmation.unit.spec.js

```diff
@@ -2,41 +2,33 @@ import React from 'react';
 import { fireEvent, render } from '@testing-library/react';
 import { expect } from 'chai';
 import sinon from 'sinon';
-
 import DisabilityConfirmation from '../../../../components/FormPages/DisabilityConfirmation';
 
 describe('hca Disability Confirmation page', () => {
-  const getData = () => ({
-    props: { data: {}, goBack: () => {}, goForward: sinon.spy() },
-  });
-
-  context('when the component renders', () => {
-    const { props } = getData();
-
-    it('should render `va-alert` with correct status', () => {
-      const { container } = render(<DisabilityConfirmation {...props} />);
-      const selector = container.querySelector('va-alert');
-      expect(selector).to.exist;
-      expect(selector).to.have.attr('status', 'info');
-    });
-
-    it('should render navigation buttons', () => {
-      const { container } = render(<DisabilityConfirmation {...props} />);
-      expect(
-        container.querySelectorAll('.hca-button-progress'),
-      ).to.have.lengthOf(2);
+  const subject = ({ goForward = () => {} }) => {
+    const props = { data: {}, goBack: () => {}, goForward };
+    const { container } = render(<DisabilityConfirmation {...props} />);
+    const selectors = () => ({
+      vaAlert: container.querySelector('va-alert'),
+      navBtns: container.querySelectorAll('.hca-button-progress'),
+      continueBtn: container.querySelector('.usa-button-primary'),
     });
+    return { container, selectors };
```

Comment:
Not a big deal and tell me if you think this DRY's it up in a good way or not, but we could destructure the selectors here in what we return like this: 

```
  return { container, ...selectors() };
```

And then when you use them you can just grab them in the subject call like this:
```
    const { vaAlert, navBtns } = subject({});
```

That eliminates the need to return the selectors and then get them. Like I said, not a huge deal but could simplify these a bit 🤷 .

Reply:
Yeah, I would keep it as is for now. I do think there is a way to refactor this to DRY up the need to call the `selectors` function as much, but it's fine for now. I do remember we had that issue with needing to ensure that it called it again. I _think_ destructuring it like that may do that still but I also may be wrong about that. I'd leave it as is for now 👍